### DISCUSSION TITLE
Security exemption enhancements - federal vs state and effective dates

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#241] [General]
In Shareworks, federal exemptions come from an enumerated list of clauses

- 3(a)(9)
- 3(a)(10)
- 4(a)(1)
- 4(a)(2)
- Rule 701
- Reg. D-504
- Reg. D-505
- Reg. D-506
- Reg. D-506(b)
- Reg. D-506(c)
- Reg. S
- Unknown

I don't know what all of these mean just wanted to bring it up for discussion...

State exemptions are free text but we also need to store the state of exemption.

Both federal and state exemptions should have an Effective Date stored along with it.

**Comment by JSv4** [comment]
Re: the enumerations:
- We are a little concerned with maintaining a list that can change due to outside factors. 
- We'd be a little more comfortable IF the "substantive issues working group" - a new working group full of lawyers - maintained this and basically gave us the definitive list at any point in time. 

Re: the jurisdiction, good point. 

Can we talk through the EffectiveDate part? Would be helpful to understand how you're tracking these and what they mean in context.
